flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/Executing.java

```diff
@@ -129,7 +131,9 @@ private void deploy() {
         for (ExecutionJobVertex executionJobVertex :
                 getExecutionGraph().getVerticesTopologically()) {
             for (ExecutionVertex executionVertex : executionJobVertex.getTaskVertices()) {
-                deploySafely(executionVertex);
+                if (executionVertex.getExecutionState() != ExecutionState.RUNNING) {
+                    deploySafely(executionVertex);
+                }
```

Comment:
Should we say that we call deploy if the `ExecutionState` is `CREATED` or `SCHEDULED`?